packages/connect-react/src/components/login/LoginInitScreen.tsx

```diff
@@ -177,7 +180,7 @@ const LoginInitScreen: FC<Props> = ({ showFallback = false }) => {
             name='email'
             label='Email address'
             type='email'
-            autoComplete='username webauthn'
+            autoComplete={sharedConfig.flags?.['conditional-ui-allowed'] !== 'false' ? 'username webauthn' : ''}
```

Comment:
As this logic is quite important, please extract it to a variable and add a comment that explains, why we need this. Also try to use hasSupportForConditionalUI() instead of accessing the flags manually if possible.